build/build.py

```diff
@@ -98,6 +98,10 @@ def __init__(self):
             '--build-for-snap', dest='build_for_snap', action='store_true',
             help='Only build components and features those required for the Ubuntu Core snap'
         )
+        self.parser.add_argument(
+            '--search-prefix', dest='search_prefix', type=str,
```

Comment:
this is not related to the snap work necessarily, right?

Reply:
Had to rebase the topic branch.